### PR TITLE
GraphRAG demo with SwarmAgents

### PR DESCRIPTION
**Pull request by AgentGenie**
## Why are these changes needed?

FalkorDB GraphRAG demo with updates to the underlying FalkorDB GraphRAG capability.

## Related issue number

N/A

## Checks

- [X] I've included any doc changes needed for https://ag2ai.github.io/ag2/. See https://ag2ai.github.io/ag2/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [X] I've made sure all auto checks have passed.

### CODE REVIEW

File: autogen/agentchat/contrib/graph_rag/falkor_graph_query_engine.py [hunk 1]
```diff
@@ -53,8 +54,30 @@ def __init__(
...
+        falkordb = FalkorDB(host=self.host, port=self.port, username=self.username, password=self.password)
+        if self.name in falkordb.list_graphs():
+            self.knowledge_graph = KnowledgeGraph(
+                name=self.name,
+                host=self.host,
+                port=self.port,
+                username=self.username,
+                password=self.password,
+                model_config=self.model_config,
+                ontology=self.ontology,
```
Comment (AgentGenie) [inline autogen/agentchat/contrib/graph_rag/falkor_graph_query_engine.py:72]: This could be None, right?

Comment (AgentGenie) [reply autogen/agentchat/contrib/graph_rag/falkor_graph_query_engine.py:77]: Updated to save ontology to the DB

**Comment by qingyun-wu** [comment]
@AgentGenie could you fix the doc formatting issues? Thank you!

**Review by marklysze — APPROVED** [review]
Notebook works with underlying FalkorDB classes, nice work!

**Comment by marklysze** [comment]
Adding metadata to notebook.

**Pull request merged**